### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Introduces GitHub Actions
- Adds, changes and deletes some project files

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017, 2020 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
+Copyright (c) 2017, 2020-2021 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io>
```
Comment (phated) [inline LICENSE]:
```suggestion
Copyright (c) 2017, 2020-2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```

File: package.json [hunk 2]
```diff
@@ -23,16 +23,17 @@
...
-    "eslint-config-gulp": "^5.0.0",
-    "expect": "^26.0.1",
-    "jest-mock": "^26.0.1",
-    "mocha": "^7.1.2",
-    "nyc": "^15.0.1",
+    "eslint": "^7.32.0",
+    "eslint-config-gulp": "^5.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "expect": "^27.4.2",
+    "jest-mock": "^27.4.2",
```
Comment (phated) [inline package.json]: Do you think we should standardize on `sinon` like the other projects?

**Comment by sttk** [comment]
@phated I've changed to use `sinon` for test mocks.

**Review by phated — APPROVED** [review]
Excellent! Thanks @sttk 🎉

**Pull request merged**